gax-java/gax-bom/pom.xml

```diff
@@ -48,12 +49,26 @@
         <groupId>com.google.api</groupId>
         <artifactId>gax</artifactId>
         <version>2.23.3-SNAPSHOT</version><!-- {x-version-update:gax:current} -->
+        <type>test-jar</type>
         <classifier>testlib</classifier>
       </dependency>
+      <dependency>
+        <groupId>com.google.api</groupId>
+        <artifactId>gax</artifactId>
+        <version>2.23.3-SNAPSHOT</version><!-- {x-version-update:gax:current} -->
+        <classifier>testlib</classifier>
```

Comment:
Adding an additional entry that does not carry "test-jar".

Reply:
@suztomo thanks for the help.